docs/telemetry.md

```diff
@@ -1,99 +1,157 @@
 
-By default, the telemetry services are disabled. To enable, set `ENABLE_OPEN_TELEMETRY=true` in `$GRAPHISTRY_HOME/data/config/telemetry.env`.
+![grafana-import-dcgm-dashboard-6](./img/grafana-import-dcgm-dashboard-6.png)
 
-The `./release` script in $GRAPHISTRY_HOME will automatically start those services when `ENABLE_OPEN_TELEMETRY=true`.  The `./release` script is anologous to `docker compose ...`, but will not trigger telemetry collection, forwarding, nor telemetry UIs unless ENABLE_OPEN_TELEMETRY is set to true and the services are properly configured. 
+## How to Use the Telemetry Data
+
+When enabled, telemetry sent to the local Graphistry [OTEL Collector](https://opentelemetry.io/docs/collector/) service, and from there, dispatches to any observability tool that is compatible with the OpenTelemetry standard (e.g. Prometheus, Jaeger, Grafana Cloud, etc.).
+
+To use telemetry with Graphistry, you need to:
 
-To start the otel services, run: 
+* Enable telemetry collection recording in one or more Graphistry services
+* Configure which users each service will monitor (see section on feature flags)
+* Enable the telemetry collector service
+* Configure the forwarding from the telemetry collector to an observability database/UI
+* Optionally, setup a local observability database/UI
+
+### Accessing Traces
+
+1. **Jaeger Dashboard:** View and analyze traces at `http://$GRAPHISTRY_HOST:16686`.
+2. **Key Tracing Information:**
+   - List of traces generated by the system for the graph rendering flow (for instance: show the trace list including a trace with errors).
+![List of traces generated by the system for the graph rendering flow](./img/jaeger-trace-list-including-trace-with-error.png)
+   - The root span for the graph rendering flow is `streamgl-viz: handleFalcorSocketConnections`.
+   - The service that generates the root span for the graph rendering flow is `streamgl-viz`.
+   - Detailed spans for actions by the visualization service and GPU workers (for instance: inspecting trace with error).
+![Detailed spans for actions by the visualization service and GPU workers](./img/jaeger-inspecting-trace-with-error.png)
+   - ETL dataset fetch spans from the Python ETL Service service.
+
+### Accessing Metrics
+1. **Prometheus Dashboard:** Access metrics at `http://$GRAPHISTRY_HOST:9091`.
+2. **Critical Metrics to Monitor:**
+   - `worker_read_crashes_total`: Monitor GPU worker crashes.
+   - File upload and dataset creation metrics in the Python ETL service (all metrics with the name `forge_etl_python_upload_*`, for instance: `forge_etl_python_upload_datasets_request_total`).
+![File upload and dataset creation metrics in the Python ETL service](./img/prometheus-forge-etl-python-metric-example.png)
+
+## Advanced configuration
+
+### Caddyfile - reverse proxy set up
+
```

Comment:
Did this file make it into the release? I didn't check, but I recall Leo saying that it should go into the other Caddyfile.default or something. But I think it might be better as a separate file for clarity.

Reply:
Yes it was part of the deployment, and in the next version this will be part of the official/standard template for Caddy.